app/assets/stylesheets/pages/track-page.scss

```diff
@@ -204,6 +176,37 @@ body.controller-my-tracks.action-show {
             }
         }
     }
+
+    .code-sample {
```

Comment:
Why has this changed scope?

Reply:
I changed it to prevent the white background color from being applied in dark theme mode.
![Screenshot from 2019-10-16 21-37-45](https://user-images.githubusercontent.com/1254212/66971045-ae652600-f05d-11e9-8c28-3153ab6795d5.png)

Reply:
But why did that fix it?

Reply:
Nesting those selectors under the 'about-section' class made them more difficult to override. As far as I can tell it was unnecessary to nest them so the best solution seemed to be to undo the nesting.

Reply:
The problem is that that is making a 38 LOC change to changing the precidence of two lines of CSS code. That makes it much harder to reason about and review the changes. It feels like things are very brittle if changing indenting levels fix them.

Would it not be better to remove the two lines of CSS that are being conditionally overriden?

Reply:
In my mind, undoing the nesting is less problematic than deleting code.

Reply:
I disagree.

If there is an underlying problem that causes issues (in this case clashing precidences) then should be fixed, otherwise we just open ourselves up for more bugs and issues later.

In this case, we have a background colour being set in one place and overridden in another. That, to me, is a code smell. And so it would be much better to fix the whole issue of the fact that the default isn't set correctly and that the overriden isn't considered in the same place.

I've built on this to achieve this with #790.

Reply:
The code in its current state is brittle because of the unnecessary nesting, which leads to conflicting styles. My change would have made it less brittle by getting rid of the unnecessary nesting. I was trying to solve the problem without changing the code more than I had needed to.

Reply:
I believe the problem is that when it's nested to some levels there is a different side effect to when it is nested to another level. The nesting itself was coincidently wrong and now we're making it work in this situation, but that just solves the symptom, it doesn't solve the problem. Which means it'll be really easy for that same problem to get introduced somewhere else later, or for us to change the nesting level on the solution css file, and suddenly that'll have precedence over this again.